.github/workflows/build.yml

```diff
@@ -4,80 +4,80 @@ on: [push, pull_request]
 
 env:
   QUAY_PATH: quay.io/brancz/kube-rbac-proxy
-  go-version: '1.22.4'
-  kind-version: 'v0.20.0'
+  go-version: "1.23"
+  kind-version: "v0.20.0"
 
 jobs:
   check-license:
     runs-on: ubuntu-latest
     name: Check license
     steps:
-    - uses: actions/checkout@v2
-    - run: make check-license
+      - uses: actions/checkout@v2
+      - run: make check-license
```

Comment:
In yaml it's recommended to use zero-indented sequences: https://www.yaml.info/learn/bestpractices.html#format

This applies to numerous changes in this file.